src/CosmosDB/CosmosDB/CosmosDBAccount/NewAzCosmosDBAccount.cs

```diff
@@ -25,7 +25,7 @@
 
 namespace Microsoft.Azure.Commands.CosmosDB
 {
-    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBAccount", DefaultParameterSetName = NameParameterSet, SupportsShouldProcess = true), OutputType(typeof(PSDatabaseAccount))]
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "CosmosDBAccount", DefaultParameterSetName = NameParameterSet, SupportsShouldProcess = true), OutputType(typeof(PSDatabaseAccount), typeof(void))]
```

Comment:
Change of output type is also considered as breaking change

output type should also be changed in help file.

Reply:
Reverted the change, to avoid Breaking changes in this PR. Would change this in a subsequent PR, so that the release in upcoming cycle is not blocked for the more important changes.